### PR TITLE
ILS: B3 version of dashboard report

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo @biyeun 
This PR fixes a problem with style in dashboard report and additionally I switched it to B3. For now B3 version is hidden in a menu to show this to Mike.

### CODE REVIEW

File: custom/ilsgateway/templates/ilsgateway/bootstrap3/datespan.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+<script type="text/javascript">
+    $(function () {
+        ko.bindingHandlers.select2 = {
+            init: function(element, valueAccessor) {
+
+                 var options = ko.toJS(valueAccessor()) || {};
+                  setTimeout(function() {
+                      $(element).select2(options);
+                  }, 0);
+               //$(element).select2(valueAccessor());
```
Comment (gcapalbo) [inline custom/ilsgateway/templates/ilsgateway/bootstrap3/datespan.html]: was this meant to be commented out? i think we should remove the line if it's never going to be used

**Comment by gcapalbo** [comment]
just one comment from me, will let @biyeun review and sign off

**Comment by biyeun** [comment]
@kkrampa can you add a screen shot?

**Comment by biyeun** [comment]
the code looks fine

**Comment by kkrampa** [comment]
ils1
ils2

**Comment by kkrampa** [comment]
@biyeun @gcapalbo Updated

**Comment by gcapalbo** [comment]
looks good to me, will let @biyeun sign off

**Pull request merged**